packages/discord.js/src/index.js

```diff
@@ -83,6 +83,7 @@ exports.ReactionManager = require('./managers/ReactionManager');
 exports.ReactionUserManager = require('./managers/ReactionUserManager');
 exports.RoleManager = require('./managers/RoleManager');
 exports.StageInstanceManager = require('./managers/StageInstanceManager');
+exports.SubscriptionManager = require('./managers/SubscriptionManager');
```

Comment:
You're using `exports.<class>` syntax in the file, you need to either deconstruct the class name from the returned value from require (`require().class`), or move to `module.exports`